docs/api.md

```diff
@@ -225,6 +226,33 @@ Response:
 }
 ```
 
+### `DELETE /api/user`
+
+Removes a user's account.
+
+#### Request headers
+
+- `GOVUK-Account-Session`
+  - the user's session identifier
+
+#### Response codes
+
+- 401 if the session identifier is invalid
+- 404 if the user cannot be found
+- 204 otherwise
+
+#### Example request / response
+
+Request (with gds-api-adapters):
+
+```ruby
+GdsApi.account_api.delete_user(
+    govuk_account_session: "session-identifier",
+)
+```
+
+Response is status code only.
+
```

Comment:
The endpoint is actually `DELETE /api/oidc-users/:subject_identifier`, and it doesn't take a `GOVUK-Account-Session` param

Reply:
Looks like I forgot to add the `UPDATE /api/oidc-users/:subject_identifier` endpoint to this file when that was introduced, I'll open a PR for that